runtime-modules/proposals/codex/src/tests/mod.rs

```diff
@@ -802,7 +802,7 @@ fn set_default_proposal_parameters_succeeded() {
         // nothing is set
         assert_eq!(<SetValidatorCountProposalVotingPeriod<Test>>::get(), 0);
 
-        ProposalCodex::set_default_config_values();
+        ProposalCodex::set_config_values(p);
 
         assert_eq!(
```

Comment:
I noticed while working on this PR that not all the config values are actually checked.

Reply:
I'll create a fix for the Nicaea.